searchlib/src/vespa/searchlib/features/queryterm.h

```diff
@@ -51,10 +61,20 @@ class QueryTermFactory {
      * @param lookupSignificance whether we should look up the significance for this term.
```

Comment:
This parameter is removed, please update

Reply:
Fixed